driver-reactive-streams/src/main/com/mongodb/reactivestreams/client/MongoCollection.java

```diff
@@ -721,8 +721,7 @@
     Publisher<Success> insertOne(ClientSession clientSession, TDocument document, InsertOneOptions options);
 
     /**
-     * Inserts a batch of documents. The preferred way to perform bulk inserts is to use the BulkWrite API. However, when talking with a
-     * server &lt; 2.6, using this method will be faster due to constraints in the bulk API related to error handling.
+     * Inserts a batch of documents. The preferred way to perform bulk inserts is to use the BulkWrite API.
```

Comment:
I don't know any longer what that second sentence means.  Using insertMany is just fine, and equivalent to using bulkWrite.  Just remove the second sentence from all the javadoc

Reply:
Done.